File: integration_tests/connector/php/ConnectTest.php
<?php
$host = $argv[1];
$username = $argv[2];
$password = $argv[3];
$database = 'test';
$port = 4000;
$ca_root_path = $argv[4];

$mysqli = mysqli_init();
$mysqli->ssl_set(NULL, NULL, $ca_root_path, NULL, NULL);
$mysqli->real_connect($host, $username, $password, $database, $port);

$result = $mysqli->query('SHOW DATABASES');
while ( $data = $result->fetch_array() ) {
      echo $data[0];
      echo "\n";
}
?>

File: integration_tests/connector/ruby/ConnectTest.rb
# Requires mysql2 package ('gem install mysql2'). Please check https://github.com/brianmario/mysql2#readme for install guide.

#!/usr/bin/ruby
require 'mysql2'

begin

    client = Mysql2::Client.new(
        host: ARGV[0],
        username: ARGV[1],
        password: ARGV[2],
        port: 4000,
        database: 'test',
        ssl_mode: :verify_ca,
    )

    result = client.query('SHOW DATABASES')
    p result.first

    client.close if client
end